test/js/calc-length.js

```diff
@@ -66,11 +66,21 @@ suite('CalcLength', function() {
     assert.strictEqual(multipliedCLenght.cssString, 'calc(40px+12.8em)');
   });
 
-  test('Multiplication of a decimal number produses expected result', function() {
+  test('Multiplication by a decimal number produses expected result', function() {
     var cLength = new CalcLength({px: 10, em: 3.2})
     var multipliedCLenght = cLength.multiply(0.5);
```

Comment:
Unfortunately the style is not to contract variable names, so these should be

calcLength
multipliedCalcLength

or some other name that uses full words. Do you mind fixing up the other places in this file while you're here?

Thanks!